go.mod

```diff
@@ -47,10 +47,11 @@ require (
 	github.com/longhorn/backing-image-manager v1.4.0-rc1.0.20230521151917-38ff27cc2cbb
 	github.com/longhorn/backupstore v0.0.0-20230612062152-f4e0de6bc37a
 	github.com/longhorn/go-iscsi-helper v0.0.0-20230529082528-4c3270590712
-	github.com/longhorn/go-spdk-helper v0.0.0-20230610063427-757b34a43c07
+	github.com/longhorn/go-spdk-helper v0.0.0-20230614144600-60ea9f38b2eb
 	github.com/longhorn/longhorn-engine v1.4.0-rc1.0.20230612073917-0896a6f78315
-	github.com/longhorn/longhorn-instance-manager v1.4.0-rc1.0.20230613094429-85c7d1d7f70b
+	github.com/longhorn/longhorn-instance-manager v1.4.0-rc1.0.20230615075334-e59ad7e20d07
 	github.com/longhorn/longhorn-share-manager v1.4.0-rc1.0.20230531125701-c996c5a415b6
+	github.com/longhorn/longhorn-spdk-engine v0.0.0-20230615022458-774de44673fa
```

Comment:
Update to the latest instead? 0305d53a3a75a8b37ed3b8a0154ecb403586f44c

Reply:
0305d53a3a75a8b37ed3b8a0154ecb403586f44c  is for DiskCreate issue. Will handled in it later.